pages/services/elastic/2.5.0-6.3.2/limitations/index.md

```diff
@@ -10,43 +10,26 @@ render: mustache
 
 ## Configuration via elasticsearch.yml and/or Elastic APIs
 
-Elasticsearch provides two ways of updating settings: persistent (through `elasticsearch.yml` file) and transient (through Elastic Settings Update API). The service's Configuration Options are carried over to the tasks' `elasticsearch.yml` file automatically. Out-of-band configuration changes (either via Elasticsearch's Update API or externally modifying `elasticsearch.yml` files) will not persist in case of a restart, failure recovery, or upgrade.
+Elasticsearch provides two ways of updating settings: persistent (through `elasticsearch.yml` file) and transient (through [Elasticsearch Cluster Update Settings API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-update-settings.html)). The service's Configuration Options are carried over to the tasks' `elasticsearch.yml` file automatically. Out-of-band configuration changes (either via Elasticsearch's Cluster Update Settings API or externally modifying `elasticsearch.yml` files) will not persist in case of a restart, failure recovery, or upgrade.
+
+## Kibana configured with X-Pack Security enabled
+
+If Kibana is configured with `kibana.elasticsearch_xpack_security_enabled` set to `true` the default DC/OS service link (`https://<cluster-url>/service/<kibana-service-name>`) will not work. This is due to a change in how Kibana deals with `Authorization` HTTP headers starting in version 6.3.
+
+As a workaround, you should be able to expose Kibana using [EdgeLB](https://docs.mesosphere.com/services/edge-lb/) by following the following how-to: [Expose Kibana using EdgeLB](/services/elastic/2.5.0-6.3.2/how-to-guides#expose-kibana-using-edgelb).
 
 #include /services/include/limitations.tmpl
 #include /services/include/limitations-zones.tmpl
 #include /services/include/limitations-regions.tmpl
 
 ## Upgrades and configuration updates
 
-Upgrades and rolling configuration updates do not wait for a green status. During deployment and upgrades, the `serial` strategy does not wait for the Elastic service to reach green before proceeding to the next node.
+Upgrades and rolling configuration updates do not wait for a cluster green health status. During deployment and upgrades, the `serial` strategy does not wait for the Elasticsearch cluster to reach green health before proceeding to the next node.
 
 ## Security
 
-Elastic's native authentication and authorization mechanisms are not supported at this time.
+Elasticsearch's native authentication and authorization mechanisms are not *officially* supported at this time.
```

Comment:
Technically, shouldn't this be: 
```Elasticsearch's native authentication and authorization mechanisms cannot be configured when the package is being installed``` 
We _do_ allow users to configure AuthN+Z with X-Pack _after_ the installation is complete.

Reply:
Right. I tried to improve on the existing paragraph, but yours is an improvement on my improvement. I'll add that.

Reply:
https://github.com/mesosphere/dcos-docs-site/commit/2804d3f865c72cfb3f2880704f8bcaa7725c3d79